tests/sclT0009-unused-symbols/sym01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:
```

Comment:
Congrats - this is cool!

Tests are falling because there is abs path of your home + working directory in reference file.
Now for that problem, there are few options:
 1) Modify Abandon, so that it won't print abs path (not so good)
 2) Modify DirSuite a little bit more: https://github.com/sn127/utils/blob/master/docs/dirsuite.md#selecting-validator

So you can overload / provide specialized selectValidator which can do fuzzy validation, and ignore changing path.  These are really simple functions, hard part will be to implement fuzzy-validator:

https://github.com/sn127/utils/blob/master/testing/src/main/scala/fi/sn127/utils/testing/DirSuiteLike.scala#L284

https://github.com/sn127/utils/blob/master/testing/src/main/scala/fi/sn127/utils/testing/TestValidator.scala